tests/listener_test.cc

```diff
@@ -241,6 +245,35 @@ TEST(listener_test, listener_bind_ephemeral_v6_port)
     ASSERT_TRUE(true);
 }
 
+TEST(listener_test, listener_bind_unix_domain)
+{
+    // Avoid name conflict by binding within a fresh temporary directory.
+    using charArray        = char[];
+    charArray nameTemplate = "/tmp/bind_test_XXXXXX";
+    auto tmpDir            = ::mkdtemp(nameTemplate);
+    ASSERT_TRUE(tmpDir != nullptr);
+    auto ss = std::stringstream();
+    ss << tmpDir << "/unix_socket";
+    auto sockName = ss.str().c_str();
+
+    struct sockaddr_un sa;
```

Comment:
```suggestion
    struct sockaddr_un sa = {};
```

Reply:
Fixed.

It turns out that line 257 was the cause of the test failures. The call to str() created a temporary that the c_str() call depended on.  But the temporary didn't survive long enough.

Reply:
We currently don't use `std::filesystem::path` in our unit tests, but you're certainly welcome to if you think it will help.